.gitignore

```diff
@@ -1,4 +1,31 @@
-.idea/
+*.class
 *.iml
-target/
-**\.DS_Store
+*.idea
+.settings
+.project
+*.sass-cache
+target
+.java-version
+*.releaseBackup
+release.properties
+README.html
+
+# Package Files #
+*.jar
+*.war
+*.ear
+*.iml
```

Comment:
*.iml is already present at line number 2

Reply:
Thanks. Sorry I have missed that. I have amended the commit and force pushed. Should be ok now.